NXP/MCIMX8M_EVK_4GB/MCIMX8M_EVK_4GB.dsc

```diff
@@ -180,4 +180,3 @@
   # Use system default resolution
   gEfiMdeModulePkgTokenSpaceGuid.PcdVideoHorizontalResolution|0
   gEfiMdeModulePkgTokenSpaceGuid.PcdVideoVerticalResolution|0
-
```

Comment:
Generally, unless there's a reason for a change, I find it's best to not have files touched unnecessarily.